plugin/core/views.py

```diff
@@ -193,6 +193,12 @@ def text_document_identifier(view_or_file_name: Union[str, sublime.View]) -> Dic
     return {"uri": uri}
 
 
+def first_selection(view: sublime.View) -> Optional[sublime.Region]:
+    if len(view.sel()):
+        return view.sel()[0]
+    return None
```

Comment:
Why not put this in a try-catch block? This might still raise IndexError at the second `view.sel()[0]` call if running in the worker thread.